shared/us-east-1/tools-prometheus/variables.tf

```diff
@@ -116,7 +116,7 @@ variable "name" {
 #
 variable "aws_ami_os_id" {
   description = "AWS AMI Operating System Identificator"
-  default     = "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*"
+  default     = "ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"
```

Comment:
@fgauchat-binbash Fran I think we're still favoring Ubu 20.04. But if you have you fully tested the associated ansible playbook (https://github.com/binbashar/le-ansible-infra/tree/master/prometheus) and prometheus resulting functionalities with this 22.04 version I think we're good to go.

CC: @diego-ojeda-binbash

Reply:
Understood, version reverted to 18.04, since 20.04 not fully tested. We will include this upgrade in future revisions.